### PR TITLE
make crowdloan duration shorter

### PR DESCRIPTION
**Pull request by GopherJ**
close: #969

### CODE REVIEW

File: launch/config.ts [hunk 1]
```diff
@@ -180,7 +181,8 @@ const config = {
       image: 'parallelfinance/polkadot-collator:v0.9.12',
       chain: 'shell',
       ctokenId: 4000,
-      cap: '100000000000000000',
+      cap: '100000000000000',
```
Comment (i5possible) [inline launch/config.ts:184]: Why this cap is changed?

File: launch/index.ts [hunk 2]
```diff
@@ -189,17 +189,17 @@ async function relay() {
 
   console.log('Start new auction.')
   const call = []
-  call.push(api.tx.sudo.sudo(api.tx.auctions.newAuction(1000000, config.leaseIndex)))
+  call.push(api.tx.sudo.sudo(api.tx.auctions.newAuction(config.auctionDuration, config.leaseIndex)))
```
Comment (i5possible) [inline launch/index.ts:192]: Change from 1000000 to 201600? Is that means 56 minutes？

File: launch/config.ts (hunk 1 above)
Comment (GopherJ) [reply launch/config.ts:184]: @i5possible because we would like to make it easier for testing the edge case

Comment (i5possible) [reply launch/config.ts:184]: So we are not goint to deploy it to prod and just for testing usage?

Comment (GopherJ) [reply launch/config.ts:184]: this script is only made for testing purpose because on prod we cannot control relaychain at all. It's up to parity

**Pull request merged**